### ISSUE TITLE
JSON schema milling enhancement

### ISSUE THREAD

**Issue by andrewxhill** [textileio/go-textile#773]
I have a JSON schema like this,

```
{
  document: Object,
  blocks: Object,
  inlines: Object,
  rules: Array,
}
```

A really nice enhancement we could handle with mills here is to not store a single encrypted leaf for this schema, but store at least 4. This would allow for much nicer dedup.

**Comment by carsonfarmer** [comment]
Closing as 'wont' fix', schema milling is no longer relevant in Threads v2
